VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts

```diff
@@ -338,7 +338,7 @@ export class ParticipantsPanelComponent implements OnInit, OnDestroy {
         }
 
         this.videoControlService.setSpotlightStatus(
-            this.participantsService.getParticipantOrVirtualMeetingRoomById(panelModel.id),
+            this.participantsService.getParticipantOrVirtualMeetingRoomById(panelModel.id, this.participants),
```

Comment:
Did you need to pass the panels to accurately get the participants. The participants service should also be up to date for this. But perhaps there's an issue with that. If so we should probably have some way to create the participant that videoControlService wants. Or we could add a method to that to take id and pexip id only. (will we have pexipId without using participants service?)

Reply:
Yep, there is a mismatch there and the list of participants is not updated if we call that directly from the participant service.
We still need their pexip properties in the panel.comp when the participants gets updated after last minute participants get added to the hearing